stable/kube-state-metrics/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 0.16.0
+version: 0.16.1
```

Comment:
this chart is in the stable folder, lets bump that to 1.0.0

Reply:
done